Rock.Messaging.RabbitMQ/IRabbitSessionConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Rock.Messaging.RabbitMQ
+{
+    public interface IRabbitSessionConfiguration
```

Comment:
re: Setters. Noted. ;-)

re: vHost. Ok, just looking for consistency. vHost it is.